### PR TITLE
React2 week2

### PR DESCRIPTION
**Pull request by na-talia**

### CODE REVIEW

File: src/client/components/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import Meal from "./Meal";
+import Title from "./Title";
+import MealsCSS from "./UI/meals.module.css";
+import MainCSS from "./UI/main.module.css";
+import { Link } from "react-router-dom";
+
+const MealLimit = () => {
```
Comment (shpomp) [inline src/client/components/HomePage.js]: Just about naming - the file says HomePage, but the component is MealLimit, while semantically meal limit is a number. 
Additionally, a Home page might get something else in there in the future. 
So, it would be nice to reconsider this a bit. The code itself is neat though  👌🏽

File: src/client/components/ReviewForm.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      })
+      .catch((error) => {
+        alert(error);
+      })
+      .finally(alert(`You review on meal № ${mealId} was sent`));
+  };
+
+  useEffect(() => {
+    (async () => {
+      await fetch(`api/meals/${mealId}`).then((response) => response.json());
```
Comment (shpomp) [inline src/client/components/ReviewForm.js:46]: Looking at it and thinking why is this needed 🤔 
You have the meal id, so you can just use it.

File: src/client/components/ReservationForm.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      })
+      .catch((error) => {
+        alert(error);
+      })
+      .finally(alert(`You have reserved a meal № ${mealId}`));
+  };
+
+  useEffect(() => {
+    (async () => {
+      await fetch(`api/meals/${mealId}`).then((response) => response.json());
```
Comment (shpomp) [inline src/client/components/ReservationForm.js:49]: Same puzzle with the meal id 🤔

**Review by shpomp — APPROVED** [review]
Very nice! 🤩

File: src/client/components/ReviewForm.js (hunk 2 above)
Comment (na-talia) [reply src/client/components/ReviewForm.js:46]: I have decided to use a fetch here so that when writing a review, the user doesn’t have to write the ID of the meal by themselves, it is already included in the form. Also the alert shows the message with the correct meal ID.

File: src/client/components/HomePage.js (hunk 1 above)
Comment (na-talia) [reply src/client/components/HomePage.js]: I have changed this file several times, added and deleted some parts, that`s why the naming could be a little bit weird, but I will change it a bit.

**Comment by na-talia** [comment]
> Very nice! 🤩

Hi Karo, Thank you so much for your review :)
